packages/api/cms-api/src/generator/utils/test-helper.ts

```diff
@@ -1,41 +1,26 @@
-import { ESLint } from "eslint";
+import * as process from "node:process";
+
+import { format, type Options, resolveConfig } from "prettier";
```

Comment:
ESM ESLint, in combination with jest, is incredibly slow. I looked into this topic in detail with @kaufmo, and we could not have a suitable performance for the tests with ESlint 9.
 
Also, there was no need to lint the specific files for the test cases. Only one test needed a properly formatted (keyword: duplicated imports) file. Prettier was added, which handles simple formatting.

Test Duration is now faster, and takes approximately 25 seconds per run.